corehq/apps/hqwebapp/tests/test_default_landing_page.py

```diff
@@ -15,11 +17,41 @@
     UserRole,
     WebUser,
 )
-from corehq.util.test_utils import flag_enabled, generate_cases
+from corehq.util.test_utils import generate_cases
 
 DOMAIN = 'temerant'
 
 
+@contextmanager
+def enable_privilege(privilege_name):
```

Comment:
Could `corehq.util.test_utils.privilege_enabled` be used here instead?

Reply:
(No, for the same reason as above.)